flexbin.scss

```diff
@@ -10,21 +10,26 @@ $flexbin-phone-max: 400px !default;
 $flexbin-row-height-phone: 100px !default;
 $flexbin-space-phone: $flexbin-space !default;
 
+$flexbin-add-end-space: false;
+
 // Minxin for customize multiple Flexbin layout
 // For example, 300px height for desktop devices, 150px height for mobile devices
 @mixin flexbin($row-height, $space) {
     display: flex;
     overflow: hidden;
     flex-wrap: wrap;
-    margin: - $space / 2;
+    margin: 0 (-1 * $space / 2);
+    padding: 2.5px 0;
```

Comment:
Do not use hard coded sizes other than `0`. Make it a variable with a proper name.